scripts/tioga.flux.sh

```diff
@@ -20,7 +20,7 @@ export PYOPENCL_CTX="AMD:0"
 
 run_cmd="flux run -N $nnodes -n $nproc --exclusive"
 
-export XDG_CACHE_ROOT="/tmp/$USER/xdg-scratch"
-export POCL_CACHE_ROOT="/tmp/$USER/pocl-cache"
+export XDG_CACHE_ROOT=${XDG_CACHE_HOME:-"$(pwd)/.mirge-cache/xdg-cache"}
```

Comment:
Should this be

```suggestion
export XDG_CACHE_ROOT=${XDG_CACHE_ROOT:-"$(pwd)/.mirge-cache/xdg-cache"}
```
?